Add docs for Meater integration (#16129)
home-assistant/home-assistant.github.io: source/_integrations/meater.markdown

---
title: Meater Smart Meat Thermometer
description: Instructions on how to integrate the Meater Temperature Probe within Home Assistant.
ha_category:
  - Sensor

> ha_release: '2021.2'

Reviewer: ```suggestion
ha_release: 2022.5
```

ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@Sotolotl'
ha_domain: meater
---

The Meater Smart Meat Thermometer integration allows for communicating with the [Meater Temperature Probe](https://meater.com/) from Apption Labs.

## Configuration

To use the integration, your probe must be connected to the Meater Cloud service.

> To enable the integration, simply set it up via the UI.

Reviewer: Please avoid the term `simply`, while it might sound simple to you, it might be hard for somebody else (and thus condescending).

Please add a description on how to do that.

Author: Done 👍

> ## Configuration
> 
> To use the integration, your probe must be connected to the Meater Cloud service.
> 
> To enable the integration, navigate to “Configuration”, then “Integrations” and click + button in the bottom right. Select "Meater Temperature Probe" to start the configuration. When prompted, enter your account information for your Meater Cloud account. Your username is usually the email address you used to sign up.

Reviewer: ```suggestion
{% include integrations/config_flow.md %}
```

---
title: Meater Smart Meat Thermometer
description: Instructions on how to integrate the Meater Temperature Probe within Home Assistant.
ha_category:
  - Sensor
ha_release: '2021.2'
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@Sotolotl'
ha_domain: meater
---

> ## Installation

Reviewer: ```suggestion
## Configuration
```

> The `meater` sensor platform is an integration for communicating with the [Meater Temperature Probe](https://meater.com/) from Apption Labs.

Reviewer: Move this line to the introduction of the integration. (a bit up).

```suggestion
The Meater Smart Meat Thermometer integration allows for communicating with the [Meater Temperature Probe](https://meater.com/) from Apption Labs.
```

To use the integration, your probe must be connected to the Meater Cloud service.

To enable the integration, either set it up via the UI, or add the following to your `configuration.yaml` file:

```yaml
# Example configuration.yaml entry
meater:
  username: "YOUR_MEATER_CLOUD_USERNAME"
  password: "YOUR_MEATER_CLOUD_PASSWORD"
  

> ```

Reviewer: This is a new integration, we should not add YAML support in that case.

> Probes will be added as soon as they're seen by home assistant. The will be unavailable when the probes are disconnected from Meater Cloud.

Reviewer: ```suggestion
Probes will be added as soon as they're seen by Home Assistant. The will be marked unavailable when the probes are disconnected from Meater Cloud.
```

> Currently both the internal and external temperature of each probe will be displayed.

Reviewer: ```suggestion
Currently, both the internal and external temperature of each probe will be displayed.
```

## Troubleshooting

> Devices will only be returned from the Meater Cloud API after the following criteria is met. There may be a delay between the Meater Cloud seeing your device and it being returned by the API.

Reviewer: ```suggestion
Devices will only be returned from the Meater Cloud API after the following criteria are met. There may be a delay between the Meater Cloud seeing your device and it being returned by the API.
```

> * Device must be seen by the Meater Cloud. Ensure you've completed a cook while connected to Meater Cloud.
> * The Meater app or Block must have an active Bluetooth connection with the device.
> * The Meater app or Block must have an active Meater Cloud connection.

Reviewer: ```suggestion
- Device must be seen by the Meater Cloud. Ensure you've completed a cook while connected to Meater Cloud.
- The Meater app or Block must have an active Bluetooth connection with the device.
- The Meater app or Block must have an active Meater Cloud connection.
```
